app/Console/Commands/ParseTextFileCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Report;
+use App\Document;
+use Illuminate\Support\Facades\Storage;
+
+class ParseTextFileCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'parse:file {filePath}';
```

Comment:
Does this mean if you want to use this library you call something like `composer parse:file ./input.txt`?

Reply:
I think you should accept parameters here letting the user choose what tasks they want to run against the file. (stats, vocab, spelling, grammer...etc.)